doc/changelog/1.21.0-changelog.rst

```diff
@@ -607,7 +607,7 @@ A total of 581 pull requests were merged for this release.
 * `#18773 <https://github.com/numpy/numpy/pull/18773>`__: CI: Fixing typo in Azure job run
 * `#18777 <https://github.com/numpy/numpy/pull/18777>`__: DOC: update random and asserts in test guidelines
 * `#18778 <https://github.com/numpy/numpy/pull/18778>`__: MAINT: Relax the integer-type-constraint of `npt._ShapeLike`
-* `#18779 <https://github.com/numpy/numpy/pull/18779>`__: DOC: fix spelling of "reccomended" ("recommended")
+* `#18779 <https://github.com/numpy/numpy/pull/18779>`__: DOC: fix spelling of "recommended" ("recommended")
```

Comment:
This is undesirable I think--the point of this PR was to fix a typo, so the typo is actually intentional in the title, as you can see by the accompanying corrected spelling in parentheses.

Reply:
I would ask that none of the changelogs are amended, it doesn't really gain anything.

Reply:
I have reverted all the changelog changes.